src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -2178,6 +2178,19 @@ For example, load-examples.groovy which is executed by default at scheduler star
 the full deployment of proactive-examples by: populating the running Catalog, exposing as templates
 specific workflows and copying workflows dependencies into dataspaces.
 
+== Add custom third-party dependencies to a workflow classpath
```

Comment:
I suggest "to workflows' classpath"

Reply:
It is very confusing to talk about workflow classpath. As actually, workflow classpath refers to a totally different feature called "Fork Environment" !

We need to talk about node classpath.